src/translations/locales/en.json

```diff
@@ -61,15 +61,15 @@
   "app.program_name.confirm": "Are you sure that you want to change the name of this program?",
   "app.program_name.name": "Name:",
   "app.program_name.save": "Save",
+  "app.program_tags.addition": "Add",
+  "app.program_tags.placeholder": "Add tags...",
+  "app.program_tags.save": "Save",
   "app.protected_route.conduct_link": "Rovercode Code of Conduct.",
   "app.protected_route.conduct_message": "All users must follow the ",
   "app.protected_route.inappropriate_message": " If you see in appropriate behavior or feel you are being harassed, please stop using Rovercode and tell your teacher.",
```

Comment:
It seems that the translation generator didn't actually change the string. Can you manually change it in this file and `es.json`?